src/coreclr/vm/methodtable.h

```diff
@@ -2788,6 +2788,19 @@ class MethodTable
         SetFlag(enum_flag_Category_Nullable);
     }
 
+    inline BOOL IsContinuation()
+    {
+        LIMITED_METHOD_DAC_CONTRACT;
+        return GetFlag(enum_flag_IsContinuationType);
```

Comment:
Can this check whether the parent type is a `CLASS__CONTINUATION` instead?

Reply:
Yeah, that seems better, no need to add this flag